searx/static/themes/simple/src/js/main/results.js

```diff
@@ -52,6 +52,7 @@
       this.innerText = this.dataset.copiedText;
     });
 
+    const isMobile = screen.orientation.type.startsWith('portrait');
```

Comment:
I don't think we should limit the back button to mobile devices, I'd like to have this feature on desktop as well.  / AFAIK “portrait” is not a good criterion for mobiles though.